src/utils/mock.ts

```diff
@@ -0,0 +1,416 @@
+const vaccineLocationOneAvailability = {
```

Comment:
I would name this file to something more precise, `mock` could be any number of things. And this shouldn't be in the `utils` folder, this is mock data, should be under `/mocks` or else it will be included in our deployment build.